src/common/Interfaces.ts

```diff
@@ -46,6 +46,8 @@ export abstract class AnimatedView extends AnimatedComponent<Types.AnimatedViewP
 }
 
 export abstract class App {
+    supportsExperimentalKeyboardNavigation = false;
```

Comment:
Why add this flag? Is there a downside of unconditionally enabling it? RN.Mac isn't being used broadly, so as long as it works in your scenarios, I'd be OK with just enabling it rather than adding hacky flags that we'll need to remove later.

Reply:
Unfortunately this flag is not available in the public variant of RN MacOS yet, therefore if it's applied to components, prop validation will fail due to lack of the prop inside ReactNativeViewAttributes.